README.md

```diff
@@ -1,6 +1,6 @@
 <img height="200" src="https://raw.githubusercontent.com/cytomining/pycytominer/master/logo/with-text-for-light-bg.png?raw=true">
 
-# Pycytominer: Data processing functions for profiling perturbations
+# Data processing functions for profiling perturbations
```

Comment:
Minor suggestion: consider a simplification here to keep things flexible (classes are involved too 🙂 ).
```suggestion
# Data processing for profiling perturbations
```

Reply:
Thanks for noting this! #289 changes the README even more than this - I will account for changes there and modify even further to simplify more.

Reply:
addressed in c9d302e1f2a85c2aa54fbe1345130d4ab886941c